epub33/rs/index.html

```diff
@@ -1591,22 +1618,23 @@ <h2>Vocabulary Association Mechanisms</h2>
 
 				<dt id="sec-property-datatype">The <code>property</code> Data Type</dt>
 				<dd>
-					<p>A Reading System MUST use the following rules to create an IRI [[RFC3987]] from a property:</p>
+					<p>A Reading System MUST use the following rules to create a URL [[URL]] from a property:</p>
 					<ul>
 						<li>
-							<p>If the property consists only of a reference, obtain the IRI by concatenating the IRI
-								stem associated with the <a href="https://www.w3.org/TR/epub-33/#sec-default-vocab"
-									>default vocabulary</a> [[EPUB-33]] to the reference.</p>
+							<p>If the property consists only of a reference, obtain the URL by concatenating the <a
+									href="https://url.spec.whatwg.org/#concept-base-url">base URL</a> [[URL]] associated
+								with the <a href="https://www.w3.org/TR/epub-33/#sec-default-vocab">default
+									vocabulary</a> [[EPUB-33]] to the reference.</p>
 						</li>
 						<li>
-							<p>If the property consists of a prefix and reference, obtain the IRI by concatenating the
-								IRI stem associated with the prefix to the reference. If the EPUB Creator has not
-								defined a matching prefix, the property is invalid and the Reading System MUST ignore
-								it.</p>
+							<p>If the property consists of a prefix and reference, obtain the URL by concatenating the
+									<a href="https://url.spec.whatwg.org/#concept-base-url">base URL</a> [[URL]]
+								associated with the prefix to the reference. If the EPUB Creator has not defined a
+								matching prefix, the property is invalid and the Reading System MUST ignore it.</p>
 						</li>
 					</ul>
-					<p>The resulting IRI MUST be valid to [[RFC3987]]. Reading Systems do not have to resolve this IRI,
-						however.</p>
+					<p>The result MUST be valid a <a href="https://url.spec.whatwg.org/#valid-url-string">valid URL
+							string</a> [[URL]]. Reading Systems do not have to resolve this URL, however.</p>
```

Comment:
Remove the 'valid' from 'MUST be valid'

Reply:
The typo definitely needs fixing, but "valid URL string" is a formal term in the URL spec. We should keep that as it ensures validation.

Reply:
I was thinking that requiring a URL would automatically mean a "valid URL" because is an invalid URL really a URL?  But I would never tamper with formal terms, so, ok :)